day3/slice_time_image.py

```diff
@@ -21,10 +21,11 @@
 """
 
 # Add any extra imports here
-import sys
-
+import sys, os
 import nibabel as nib
```

Comment:
Typical convention is to put less common imports lower down than more common.  So, numpy is more common than scipy which is about the same popularity as matplotlib, and more common than nibabel.